sentry-ruby/lib/sentry/transport/http_transport.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "net/http"
+require "resolv"
```

Comment:
Could you add it to the `net/http` patch instead as it's required by it, not `HTTPTransport`.

Reply:
/facepalm my bad, one sec

Reply:
Done.